hw01_hello_otus/go.mod

```
@@ -1,3 +1,10 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/Lanworm/OTUS_GO/tree/master/hw01_hello_otus
```

```suggestion
module github.com/Lanworm/OTUS_GO/hw01_hello_otus
```
`/tree/master` - часть роутинга UI GitHub, т.е. не имеет отношения к модулю Go